app/jobs/bulkrax/child_relationships_job.rb

```diff
@@ -78,7 +78,6 @@ def work_child_collection_parent(work_id)
         ImporterRun.find(importer_run_id).increment!(:processed_children)
       rescue StandardError => e
         entry.status_info(e)
-        entry.save!
```

Comment:
Do we still need to save `entry` here, even though we no longer need to in order to capture the status? Was the only purpose of these `save!`s to capture the error?

Reply:
it was the only reason for the save, nothing else has changed in the entry at that point.